contracts/MarginPool.sol

```diff
@@ -77,10 +76,58 @@ contract MarginPool {
         address _asset,
         address payable _user,
         uint256 _amount
-    ) external onlyController returns (bool) {
-        require(_amount > 0, "MarginPool: transferToUser amount is below 0");
+    ) public onlyController returns (bool) {
+        require(_amount > 0, "MarginPool: transferToUser amount is below or equal to 0");
 
         // transfer asset val from Pool to _user
         return ERC20Interface(_asset).transfer(_user, _amount);
     }
+
+    /**
+     * @notice transfers multiple assets from users to pool
+     * @dev all tokens are scaled to have 1e18 precision in contracts, but are scaled to native
+     *      token decimals in Controller before being passed to MarginPool
+     * @param _asset addresses of assets to transfer
+     * @param _user addresses of users to transfer assets to
+     * @param _amount amount of each token to transfer to _user, scaled to 1e18 of precision
+     */
+    function batchTransferToPool(
+        address[] memory _asset,
+        address[] memory _user,
+        uint256[] memory _amount
+    ) public onlyController {
+        require(
+            _asset.length == _user.length && _user.length == _amount.length,
+            "MarginPool: batchTransferToPool array lengths are not equal"
+        );
+
+        for (uint256 i = 0; i < _asset.length; i++) {
+            // transfer val from _user to pool
+            transferToPool(_asset[i], _user[i], _amount[i]);
```

Comment:
In standard ERC20 `transfer` and `transferFrom`, the returned boolean will indicate whether the transfer is successful (ERC20 doesn't require a revert in failed transactions, although it's widely used). 
So we shouldn't ignore the returned value from `transferToUser` and `transferToPool`.
```suggestion
            require(transferToPool(_asset[i], _user[i], _amount[i]), "MarginPool: Transfer to pool failed");
```